### ISSUE TITLE
Suggestion for join - split:

### ISSUE THREAD

**Issue by MARIOBASZ** [vlachoudis/bCNC#1083]
It is a join advantage to manipulate different blocks all together.
When doing split on join blocks, they appear in reverse order (beginning or end) It would be better if the original order was not altered.

**Comment by Harvie** [comment]
Yes, i have similar problem in almost all of my plugins. Haven't figured it out yet.
